### ISSUE TITLE
Cannot click on authorization window.

### ISSUE THREAD

**Issue by 9000-volts** [birdieapp/birdie#173]
I have just installed the app on eos via the .deb on your website.
I managed to get logged in but on the authorization screen I can not click 'authorize app', 'cancel', or any other clickable things in the browser panel.

I *can* right-click on links and click 'Open link' to follow the links, but this does not help for the 'authorize' button, unfortunately. Regular clicking anywhere on the panel triggers no reaction from the app.

**Comment by nathandyer** [comment]
Hi! Can you show me a screenshot of what it looks like for you? If you press control+q to quit the app and open it back up, does it work then?

**Comment by 9000-volts** [comment]
@nathandyer Although I had tried restarting the app several times already, it looks like it actually worked this time! (Although since it happened at all it's probably still an issue, perhaps.)

To add more detail, : when I first started it up and hit 'log in', it went to the log in screen, which was in french for some reason. After I logged in, the app quit automatically. When I started it back up, I hit 'log in' again and it took me directly to the authorization screen. This is where I had encountered the problem, but it was fixed after I tried running birdie with the `-d` flag (it still works now that I'm not running it with that flag.)
